chart/values.yaml

```
@@ -637,6 +637,12 @@ kerberos:
 
 # Airflow Worker Config
 workers:
+  # Enable the default worker defined by the root 'workers' configuration.
+  # If false, only workers defined in 'sets' will be created.
```

```suggestion
  # Enable the default workers defined by the root 'workers' configuration to be created.
  # If false, only dedicated workers defined in 'sets' will be created.
```
I feel like the description from one point is good and from the other something is missing for me, so just a small proposal